pkg/types/conversion/installconfig.go

```diff
@@ -95,12 +93,6 @@ func convertNetworking(config *types.InstallConfig) {
 		netconf.NetworkType = netconf.DeprecatedType
 	}
 
-	// Recognize the OpenShiftSDN network plugin name regardless of capitalization, for
-	// backward compatibility
-	if strings.ToLower(netconf.NetworkType) == strings.ToLower(string(operv1.NetworkTypeOpenShiftSDN)) {
-		netconf.NetworkType = string(operv1.NetworkTypeOpenShiftSDN)
-	}
```

Comment:
I think we should keep this, because later on we are comparing against `operv1.NetworkTypeOpenShiftSDN` (e.g. in `validateNetworking()`).
Removing this conversion would make it behave like a third-party plugin (e.g. Calico) if the capitalisation didn't match exactly.